.github/workflows/pr-realm-js.yml

```diff
@@ -713,6 +738,15 @@ jobs:
           arch: x86
           ndk: ${{ env.NDK_VERSION }}
           cmake: 3.22.1
-          working-directory: integration-tests/environments/react-native-test-app
-          script: npx mocha-remote -- concurrently --kill-others-on-fail npm:metro npm:runner
-          
+          working-directory: integration-tests/environments/react-native-test-app/android
+          script: |
+            # Setup port forwarding to Mocha Remote
+            adb reverse tcp:8090 tcp:8090
+            # Build and install the app
+            ./gradlew installRelease
+            # Launch the app
+            # -W wait for launch to complete, to allow the logcat script to find a pid
+            # -S force stop the target app before starting the activity
+            adb shell am start -W -S -n com.microsoft.reacttestapp/.MainActivity
```

Comment:
The use of `-S` here should also fix the issues we're having with the Android test app being unable to install on other branches: https://github.com/realm/realm-js/actions/runs/9680928508/job/26757061658?pr=6743#step:16:526
> Unable to install /Users/runner/work/realm-js/realm-js/integration-tests/environments/react-native-test-app/android/app/build/outputs/apk/debug/app-debug.apk
[runner] com.android.ddmlib.InstallException: INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package com.microsoft.reacttestapp signatures do not match previously installed version; ignoring!